DEVELOPMENT.md

```diff
@@ -88,7 +88,19 @@ This project takes a very collaborative and [agile](https://agilemanifesto.org/)
 
 Releases follow the [CalVer](https://calver.org/) versioning using a `YYYY.MM.DD-N` format. Start with `-1` for a release, incrementing if there is another release on the same day.
 
-Releases should be [created in Github](https://github.com/HHS/simpler-grants-gov/releases) and include a log of changes.
+Releases are [created in Github](https://github.com/HHS/simpler-grants-gov/releases) and with a log of changes.
+
+Step by step instructions for creating a release:
+
+- In your terminal, `git switch main` and `git pull` to sync your local code with the latest main commit
+- In your terminal, `git tag YYYY.MM.DD-N` (FILLED IN!) and `git push --tags` to tag and push your CalVer tag
+- On Github.com, open the Releases page: https://github.com/HHS/simpler-grants-gov/releases
+- Click "Draft a new release"
+- Choose the tag you just pushed as the current tag, and the last CalVer release as the previous tag
+- Click "Generate release notes" to auto-generate release notes
+- Click "Publish" to finalize this step
+- On Github.com, open the Actions page: https://github.com/HHS/simpler-grants-gov/actions
+- You should see Actions that correspond to your release. Follow their status to ensure that they succeed. Follow-up may be required if the Github Actions fail.
```

Comment:
Let's add a link here to the github actions page for deploying a release

Reply:
@SammySteiner I'm not 100% up to speed on how to Github Actions work yet, so I'm going to link the general Github actions page!

Reply:
🤦🏼‍♀️ I already did that. Please hold, I will figure out the specific Github Actions page for deploying a release 👍🏼